integration-tests/support/login.ts

```diff
@@ -11,11 +8,12 @@ declare global {
 }
 
 const KUBEADMIN_USERNAME = 'kubeadmin';
-
-// any command added below, must be added to global Cypress interface above
+const loginUsername = Cypress.env('BRIDGE_KUBEADMIN_PASSWORD')
```

Comment:
is this correct? Asking since then looking into the console elements I see that `nav` element does have the `data-test` attribute set to the `user-dropdown`, but insie it there is another `span` element with `data-test` attribute set to `username`.
wouldn't we always the match the element ?
<img width="1244" alt="Screenshot 2023-04-11 at 16 10 27" src="https://user-images.githubusercontent.com/1668218/231190313-42f200a5-a36f-450f-afc3-800f3d221948.png">

Reply:
> is this correct? Asking since then looking into the console elements I see that `nav` element does have the `data-test` attribute set to the `user-dropdown`, but insie it there is another `span` element with `data-test` attribute set to `username`. wouldn't we always the match the element ? <img alt="Screenshot 2023-04-11 at 16 10 27" width="1244" src="https://user-images.githubusercontent.com/1668218/231190313-42f200a5-a36f-450f-afc3-800f3d221948.png">

@jhadvig  The `user-dropdown`  is not for `nav` but to find the  the login `username`